### PR TITLE
Hack Theme Formatting and Repo setup

### PR DESCRIPTION
**Pull request by nishantbalaji**
* Added Hack coloring + styles to site, and redirecting theme links to ACM links
* Added CODEOWNERS file with ACM Hack team as owner
image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
+    "url": "git+https://github.com/acmucsd/nextra-docs.git"
   },
   "author": "Shu Ding <g@shud.in>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/shuding/nextra-docs-template/issues"
+    "url": "https://github.com/acmucsd/nextra-docs/issues"
   },
-  "homepage": "https://github.com/shuding/nextra-docs-template#readme",
+  "homepage": "https://github.com/acmucsd/nextra-docs#readme",
```
Comment (alexzhang1618) [inline package.json:20]: Will most likely have to rename these since the repo name was changed

File: styles.css
Comment (alexzhang1618) [inline styles.css:1]: Just curious, did you get this file from somewhere? If so, should we link it at the top of the file?

File: theme.config.tsx [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
-    text: 'ACM Blog is so cool!',
+    text: 'ACM Hack is so cool!',
   },
   search: {
     placeholder: 'Search',
   },
   sidebar: {
     toggleButton: true,
   },
+  docsRepositoryBase: 'https://github.com/acmucsd/nextra-docs/blob/main',
```
Comment (alexzhang1618) [inline theme.config.tsx:28]: Same here with renaming the repo

File: theme.config.tsx [hunk 3]
```diff
@@ -1,18 +1,31 @@
...
+      };
+    }
+    return {
+      titleTemplate: '%s',
+    };
+  },
   footer: {
-    text: 'ACM Blog is so cool!',
+    text: 'ACM Hack is so cool!',
   },
```
Comment (alexzhang1618) [inline theme.config.tsx:21]: We can open an issue to set a footer like the main website

File: .eslintrc.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
     "no-shadow": "off",
     "react/require-default-props": "off",
     "jsx-a11y/control-has-associated-label": "off",
+    "camelcase": "off",
```
Comment (alexzhang1618) [inline .eslintrc.json:19]: Is there a reason why this is turned off?

**Review by alexzhang1618 — APPROVED** [review]
Everything looks good! 
Some comments here I think we should address, but after that looks good to merge:
- The logo font is kinda small/hard to read, do we have a hack logo where the font is larger?
  - I think the hack logo for the Discord server looks better for example
Screenshot 2023-08-01 at 10 10 42 PM
Screenshot 2023-08-01 at 10 09 01 PM

File: styles.css
Comment (nishantbalaji) [reply styles.css:1]: pulled from codesandbox open source and modified

File: theme.config.tsx (hunk 3 above)
Comment (nishantbalaji) [reply theme.config.tsx:21]: maybe we should do the same for a header

File: .eslintrc.json (hunk 4 above)
Comment (nishantbalaji) [reply .eslintrc.json:19]: `import DM_Sans `

breaks

File: styles.css
Comment (alexzhang1618) [reply styles.css:1]: Cool, let's link it somewhere just in case we want to reuse it down the line (perhaps for a dev team docs site :o)

**Review by nik-dange — COMMENTED** [review]
lgtm 🚀

**Comment by nishantbalaji** [comment]
> Everything looks good! Some comments here I think we should address, but after that looks good to merge:
> 
> * The logo font is kinda small/hard to read, do we have a hack logo where the font is larger?
>   
>   * I think the hack logo for the Discord server looks better for example
> 
> Screenshot 2023-08-01 at 10 10 42 PM Screenshot 2023-08-01 at 10 09 01 PM

Regarding the logo, lets make a new PR for it. I couldn't find the svg so its not updated here

**Comment by nishantbalaji** [comment]
> lgtm 🚀

silly goose u gotta hit approve

**Pull request merged**